### PR TITLE
Update opentelemetry docs to prefer OTLP http/protobuf over grpc

### PR DESCRIPTION
**Pull request by jack-berg**
The grpc version of OTLP causes problems compared to http/protobuf. This PR updates all the OpenTelemetry docs to prefer http/protobuf. 

The OpenTelemetry docs reference examples in [newrelic/newrelic-opentelemetry-examples](https://github.com/newrelic/newrelic-opentelemetry-examples). The following PRs update all the examples in that repo to prefer http/protobuf over grpc as well:

- https://github.com/newrelic/newrelic-opentelemetry-examples/pull/524
- https://github.com/newrelic/newrelic-opentelemetry-examples/pull/523
- https://github.com/newrelic/newrelic-opentelemetry-examples/pull/522
- https://github.com/newrelic/newrelic-opentelemetry-examples/pull/521
- https://github.com/newrelic/newrelic-opentelemetry-examples/pull/520
- https://github.com/newrelic/newrelic-opentelemetry-examples/pull/519

### CODE REVIEW

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/opentelemetry-tutorial-go.mdx
Comment (jack-berg) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/opentelemetry-tutorial-go.mdx:1]: This file was odd. It doesn't appear in the navigation, but is linked to by [the metrics best practices page](https://docs.newrelic.com/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-metrics/).

The problem is that the page is wrong. It seems to be a copy / paste of the [python getting started guide](https://docs.newrelic.com/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/opentelemetry-tutorial-python/), but was never updated to actually be a go getting started guide like the title states. 

Better to delete the document for now and re-add it when someone wants to properly add a go getting started page.

cc @reese-lee who committed the page

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/opentelemetry-tutorial-java.mdx [hunk 1]
```diff
@@ -1023,12 +1024,19 @@ This is a list of the environment variables you should export if you're doing tu
         </tr>
         <tr>
           <td>
-            `OTEL_EXPORTER_OTLP_ENDPOINT=https://otlp.nr-data.net:4317` (For the US)
-            `OTEL_EXPORTER_OTLP_ENDPOINT=https://otlp.eu01.nr-data.net:4317` (For the EU)
+            `OTEL_EXPORTER_OTLP_ENDPOINT=https://otlp.nr-data.net:` (For the US)
+            `OTEL_EXPORTER_OTLP_ENDPOINT=https://otlp.eu01.nr-data.net:` (For the EU)
```
Comment (kaylareopelle) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/opentelemetry-tutorial-java.mdx:1028]:
```suggestion
            `OTEL_EXPORTER_OTLP_ENDPOINT=https://otlp.nr-data.net` (For the US)
            `OTEL_EXPORTER_OTLP_ENDPOINT=https://otlp.eu01.nr-data.net` (For the EU)
```

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/opentelemetry-tutorial-java.mdx [hunk 2]
```diff
@@ -112,10 +112,11 @@ This is a great option if you want us to do the instrumentation so you can quick
     ```
 2. Set these two environment variables to send data to your New Relic account:
     * Make sure to use your <InlinePopover type="licenseKey" />.
-    * If your New Relic data center region is EU and not US, set the endpoint to: `https://otlp.eu01.nr-data.net:4317`
+    * If your New Relic data center region is EU and not US, set the endpoint to: `https://otlp.eu01.nr-data.net`
 
       * `OTEL_EXPORTER_OTLP_HEADERS=api-key=INSERT_YOUR_NEW_RELIC_LICENSE_KEY`
-      * `OTEL_EXPORTER_OTLP_ENDPOINT=https://otlp.nr-data.net:4317`
+      * `OTEL_EXPORTER_OTLP_ENDPOINT=https://otlp.nr-data.net:`
```
Comment (kaylareopelle) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/opentelemetry-tutorial-java.mdx:118]:
```suggestion
      * `OTEL_EXPORTER_OTLP_ENDPOINT=https://otlp.nr-data.net`
```

**Review by kaylareopelle — CHANGES_REQUESTED** [review]

**Review by rhetoric101 — APPROVED** [review]
@jack-berg thanks for tackling this sweeping update! This looks good!

**Review by jeff-colucci — APPROVED** [review]
All green, looks good to me!

**Comment by jack-berg** [comment]
@jeff-colucci is there automated tooling for resolving the merge conflicts or should I take care of that?

**Comment by jeff-colucci** [comment]
@jack-berg Unfortunately, there is no automated tool that I'm aware of for branch conflicts, so they have to be resolved manually. I'm not sure where these branch conflicts came from since they weren't there when I approved the request I don't think.

File: src/i18n/content/jp/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-android-gradle-android-studio.mdx
Comment (kaylareopelle) [inline src/i18n/content/jp/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-android-gradle-android-studio.mdx:1]: @jack-berg Are these android gradle/studio files related to OpenTelemetry? Why are they being deleted?

Comment (jack-berg) [reply src/i18n/content/jp/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-android-gradle-android-studio.mdx:1]: I don't know. I didn't commit those changes [see commit [here](https://github.com/newrelic/docs-website/pull/15846/commits/50770de500e5238830066e234c6951701a9ba629)]. I assumed it was some sort of unrelated automated cleanup required to get the develop branch back healthy which was just being bundled with this PR by coincidence, but @rhetoric101 and @jeff-colucci may know more.

Comment (kaylareopelle) [reply src/i18n/content/jp/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-android-gradle-android-studio.mdx:1]: The file I commented on went away, but there are still a few others.

Comment (jeff-colucci) [reply src/i18n/content/jp/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-android-gradle-android-studio.mdx:1]: Hi guys! Thanks for fixing this. I was able to track down that there was a script that was run that deleted some orphaned translated files, and that seems to be where the discrepancy came from. I'm going to re-check the files now to make sure there are no more issues. We have an auto-translator that does our translation work for us, so if a translated doc that doesn't cause branch commits gets deleted, that translator will re-translate the files so we won't actually lose anything.

Apologies for the confusion and thanks for patiently sticking with us while we get this ticket worked on. Assuming there are no more branch conflicts I will go ahead and get this merged and we can mark it as done!

**Review by jeff-colucci — APPROVED** [review]
Everything checks out!

**Pull request merged**